charts/pulsar/values.yaml

```diff
@@ -813,6 +812,8 @@ grafana:
   ##
   service:
     type: LoadBalancer
+    port: 3000
```

Comment:
as the port is for service mapping, it would be more suitable to keep it under service section.